### ISSUE TITLE
Confirm that all of the code, data, and text is your sole IP

### ISSUE THREAD

**Issue by platipodium** [rchartra/NcCut#11]
This is part of my review at https://github.com/openjournals/joss-reviews/issues/7185

Currently, no other sources for data, text, and images are acknowledged.  So I must assume, you coded, wrote, and created all of the artifacts in your repository yourself and by yourself.  Did you really reinvent all the wheels?

Please confirm that this is the case.  If not, please 
* acknowledge your contributors
* cite your sources
* clarify licensing issues with your sources and contributors.

**Comment by rchartra** [comment]
All code, text, videos and images here and in my docs are my own work and mine alone. Some of the example data and images I provide are not mine though they come from free access sources. I can add formal citations to them in an acknowledgements section.

**Comment by rchartra** [comment]
The packages I used in my code are all open source and free access. Do these need citations as well?

**Comment by platipodium** [comment]
> The packages I used in my code are all open source and free access. Do these need citations as well?

No.  Software that you do not distribute yourself does not need to be cited or acknowledged.

**Comment by platipodium** [comment]
> All code, text, videos and images here and in my docs are my own work and mine alone. 

Perfect 👍 

> Some of the example data and images I provide are not mine though they come from free access sources.

Free access does not mean that you can use them legally.  For each of those data, you need to check the license and you need to state the license and the copyright holder, if you want to distribute that data as part of your (source code) package.   

>  I can add formal citations to them in an acknowledgements section. 

That is usually not required, except when stated in the license.  But you can - of course - be nice and acknowledge anyway.

**Comment by rchartra** [comment]
Alright so following UW's licensing recommendations I've switched over to the BSD 3 Clause license following the Reuse Software guidelines. UW indeed holds the copyright rights so I've added copyright statements indicating that. As for my non code files, my example data comes from the family of MITgcm LLC simulations which are all listed as being in the Public Domain. One of my example images comes from the ESA and I've followed their protocol laid out by their CC-BY-SA-3.0-IGO license. All my other images I created I've released under CC0-1.0.
